core/src/main/resources/org/apache/spark/ui/static/streaming-page.js

```diff
@@ -300,6 +300,156 @@ function drawHistogram(id, values, minY, maxY, unitY, batchInterval) {
     }
 }
 
+function drawAreaStack(id, labels, values, minX, maxX, minY, maxY) {
+    d3.select(d3.select(id).node().parentNode)
+        .style("padding", "8px 0 8px 8px")
+        .style("border-right", "0px solid white");
+
+    // Setup svg using Bostock's margin convention
+    var margin = {top: 20, right: 27, bottom: 30, left: maxMarginLeftForTimeline};
+    var width = 850 - margin.left - margin.right;
+    var height = 300 - margin.top - margin.bottom;
+
+    var svg = d3.select(id)
+      .append("svg")
```

Comment:
Maybe, `4 space` indent here to be consistent with the others?

Reply:
Before this line, it seems that you follows `4-space`. After this line, it seems that you uses `2-space`. It would be good if the newly added code is consistent.

Reply:
ok